test/unit/annotation_spec.js

```diff
@@ -1659,6 +1685,64 @@ describe("annotation", function () {
         done();
       }, done.fail);
     });
+
+    it("should get rendered annotation for printing", function (done) {
```

Comment:
`"should render checkboxes for printing"` (since radio buttons will be added in a next patch in another test).